### PR TITLE
Video updates

### PR DESCRIPTION
**Pull request by ryanr14**
Updates the WooCommerce 101 references to WooCommerce Guided Tour.
Slight tweak to the embed code to allow time stamp embeds.
Added some new embeds.

### CODE REVIEW

File: includes/admin/class-wc-admin-help.php [hunk 1]
```diff
@@ -38,116 +38,164 @@ public function add_tabs() {
...
+				'title' => __( 'General Settings', 'woocommerce' ),
+				'url'   => '//fast.wistia.net/embed/iframe/mz2l10u5f6.jsonp?',
 			),
 			'wc-settings-products' => array(
-				'title' => __( 'Product settings', 'woocommerce' ),
-				'url'   => '//fast.wistia.net/embed/iframe/lolkan4fxf?videoFoam=true',
+				'title' => __( 'Product Settings', 'woocommerce' ),
+				'url'   => '//fast.wistia.net/embed/iframe/lolkan4fxf.jsonp?',
+			),
+				'wc-settings-products-display' => array(
```
Comment (claudiulodro) [inline includes/admin/class-wc-admin-help.php]: The tabbing goes a little awry here. :)

**Review by claudiulodro — CHANGES_REQUESTED** [review]
Just needs a minor tabbing tweak. Videos work great.

File: includes/admin/class-wc-admin-help.php (hunk 1 above)
Comment (claudiulodro) [inline includes/admin/class-wc-admin-help.php]: I'll fix it real quick so we can get this out the door.

**Review by claudiulodro — APPROVED** [review]

**Comment by ryanr14** [comment]
Thanks a ton, @claudiulodro. :)

**Comment by claudiosanches** [comment]
This TitleCase again will make people sad.

**Pull request merged**